.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -177,7 +177,9 @@ buildvariants:
   # package
 <% for (const buildVariant of PACKAGE_BUILD_VARIANTS) { %>
   - name: <%= buildVariant.name %>
-
+    expansions:
+      # macos instead of macos-x64 since asset groups were created before build variant renaming
+      silk_asset_group: compass-<%= buildVariant.name.replace('package-', '').replace('macos-x64', 'macos') %>
```

Comment:
would it be safer to add `silk_asset_group` explicitly to `buildVariant`?

Reply:
Sure – done!